src/assets/scripts/util/autofit-images.js

```diff
@@ -20,7 +20,12 @@ var autoFitImages = {
     });
     this.loadImagesByWidth(autoFitImages.el.imageString, autoFitImages.el.backgroundString);
     this.loadIfNotVisible(0);    

```

Comment:
I want to be moving away from the heavy DOM crawling and at least moving toward a more unobtrusive approach with our theme. What does this implementation look like if we want to remove this file?